src/concatAll.js

```diff
@@ -0,0 +1,34 @@
+import { concat, identical, identity, pipe, when, reduce } from 'ramda';
+
+import stubNull from './stubNull';
+import fl from './fantasy-land/mapping';
+
+const nullSemigroup = { [fl.concat]: identity };
```

Comment:
I guess we'll need to change this to be compatible from ramda 0.19.1 to ramda 0.25.0

Reply:
Already done. Changed back to `'concat'`.